Makefile

```diff
@@ -209,3 +209,7 @@ endif
 # Help documentatin à la https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
 help:
 	@cat Makefile | grep -v '\.PHONY' |  grep -v '\help:' | grep -B1 -E '^[a-zA-Z0-9_.-]+:.*' | sed -e "s/:.*//" | sed -e "s/^## //" |  grep -v '\-\-' | sed '1!G;h;$$!d' | awk 'NR%2{printf "\033[36m%-30s\033[0m",$$0;next;}1' | sort
+
```

Comment:
While running `make` I am getting the following error: `go: github.com/ilgooz/mattermost-dlock@v0.0.0-20191107223609-c278c8c31ad6: parsing ../../../mattermost-dlock/go.mod: open /Users/Rajat_Sanjay_Varyani/go/src/github.com/mattermost/mattermost-dlock/go.mod: no such file or directory
make: build/bin/manifest: Command not found
build/setup.mk:13: *** "Cannot parse id from plugin.json".  Stop.`

Can you please check if it's happening for you as well?

Reply:
Having a simple pipeline can be the first safety net for such issues. Wdyt @ilgooz ?

Reply:
this was due to https://github.com/ilgooz/mattermost-plugin-marketplace-addon/pull/3#discussion_r346817059.

fixed by https://github.com/ilgooz/mattermost-plugin-marketplace-addon/pull/3/commits/37330de13a566c50a6fa16f0113f373db0893f71